src/ProjectTemplates/Microsoft.Extensions.AI.Templates/src/ChatWithCustomData/ChatWithCustomData-CSharp.Web/ResponseClientHelper.cs

```diff
@@ -0,0 +1,36 @@
+using Microsoft.Extensions.AI;
+using Aspire.OpenAI;
+using OpenAI;
+
+//namespace chat.aspire.Web;
+namespace Microsoft.Extensions.Hosting;
+
+public static class ResponseClientHelper
```

Comment:
```suggestion
public static class AspireOpenAIClientBuilderResponsesChatClientExtensions
```
I'm wondering if the class should be named more akin to AspireOpenAIClientBuilderChatClientExtensions, what do you think?

Reply:
I agree